src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs

```diff
@@ -87,7 +87,7 @@ internal bool Initialize(UpdateRenderPackageParams initParams)
                 throw new ArgumentNullException("initParams.DrawableIds");
 
             var nodeModel = initParams.Node;
-            if (!nodeModel.WasInvolvedInExecution && !initParams.ForceUpdate)
+            if (nodeModel.WasRenderPackageUpdatedAfterExecution && !initParams.ForceUpdate)
                 return false; // Not has not been updated at all.
```

Comment:
It should be AND. Even if the render package has been updated, we still need to update it again if it is forced to be updated.